specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json

```diff
@@ -2072,6 +2491,27 @@
       },
       "x-ms-azure-resource": true
     },
+    "Error": {
```

Comment:
this is not the right schema for error. sample here - https://github.com/Azure/azure-rest-api-specs/blob/master/specification/batch/resource-manager/Microsoft.Batch/stable/2019-08-01/BatchManagement.json#L105

Reply:
@ravbhatnagar Just submitted a new commit to solve the problem